client/webserver/api.go

```diff
@@ -189,6 +189,39 @@ func (s *WebServer) apiTrade(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, resp, s.indent)
 }
 
+// apiAccountExport is the handler for the '/account' API request.
+func (s *WebServer) apiAccountExport(w http.ResponseWriter, r *http.Request) {
+	form := new(accountExportAuthForm)
+	if !readPost(w, r, form) {
+		return
+	}
+	r.Close = true
+	accountResponse, err := s.core.AccountExport(form.Pass, form.Host)
+	if err != nil {
+		s.writeAPIError(w, "error retrieving keys: %v", err)
+		return
+	}
```

Comment:
A couple of things need tweaking here. 

1. The `AccountExport` method returns a `core.Account`, not `core.AccountResponse`, also meaning that `core.AccountResponse` is unused. 
2. **But**, you do want to have separate types account and response. `core.AccountResponse` should be a **webserver** type though, like all of the other responses.
3. After you resolve **2**, remove the `OK` field from the `core.Account`. That field is only for the API response.